test/benchee/conversion/count_test.exs

```diff
@@ -1,58 +1,63 @@
 defmodule Benchee.Conversion.CountTest do
   use ExUnit.Case
+  alias Benchee.Conversion.Count
   import Benchee.Conversion.Count
   doctest Benchee.Conversion.Count
 
+  defp unit_for(unit_atom) do
+    Map.fetch! Count.units, unit_atom
+  end
```

Comment:
I like the idea, but that would be harder to use wouldn't it? E.g. for the use case that I have some atom like `:billion` I'd have to do whatever `Count.apply my_atom` or so is in Elixir, and then I sort of prefer doing something like `unit_for` - not quite sure though :)